fixtures/cameo/ts-100-ww.json

```diff
@@ -157,17 +148,17 @@
       "channels": [
         "Dimmer",
         "Dimmer fine",
-        "Strobe"
+        "Shutter / Strobe"
       ]
     },
     {
       "name": "5ch",
```

Comment:
Can you please rename this mode to "5-channel" with short name "5ch" (and similarly for the other modes)?

Reply:
Oh right, missed this one

Reply:
Done